api/HTMLAnchorElement.json

```
@@ -284,8 +284,8 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": false,
-            "deprecated": true
+            "standard_track": true,
+            "deprecated": false
```

The IDL for this has the `// also has obsolete members` comment link. It's both standard track and deprecated.

https://html.spec.whatwg.org/multipage/text-level-semantics.html#htmlanchorelement

```suggestion
            "deprecated": true
```